eng/emitter-package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "TempTypeSpecFiles",
-  "lockfileVersion": 3,
+  "lockfileVersion": 2,
```

Comment:
I am not sure why but there is lots of diff in lock file. Is it npm version causing it?

Reply:
It should still follow your local nodejs/npm

Not sure but I guess you need to have 18/9

Reply:
I'm using 20/8. Let me update my npm to 9.

Reply:
Regen by 20/10.